.travis.yml

```diff
@@ -15,6 +15,14 @@ before_script:
 cache: bundler
 language: ruby
 rvm:
+
+##  - 2.1.5
+##  - 2.2.1
+#  - 2.3.1
+#script: bundle exec rspec --color --format documentation --require spec_helper --require rails_helper --tag ~docker
+
   - 2.1.5
   - 2.2.1
-script: bundle exec rspec
+  - 2.3.1
+script: bundle exec rspec --require spec_helper --require rails_helper --tag ~docker
```

Comment:
docker tag is necessary to pass tests using docker properly, otherwise all tests are failing. I have added color and documentation option.